platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/cache/VirtualNodeCache.java

```diff
@@ -1060,89 +1023,11 @@ public Stream<VirtualHashRecord> dirtyHashesForFlush(final long lastLeafPath) {
     /**
      * Gets estimated number of dirty hashes in this cache.
      *
-     * <p>This method and similar methods about leaves are called multiple times. First,
-     * the method is used to estimate this cache copy size to make a decision whether to
-     * flush the copy to disk or to merge it to a newer copy. If the copy is selected to
-     * flush, its {@link #prepareForFlush()} method is called, it may mark some mutations
-     * as filtered, which affects the estimated size. After that this method is called
-     * again, it takes filtered mutations into consideration. If the updated estimated
-     * size is lower than flush threshold, the copy is not flushed to disk, but rather
-     * its {@link #garbageCollect()} is called.
-     *
      * @return
      *        Estimated number of dirty hashes
```

Comment:
Estimated number of dirty internal nodes

Reply:
"Internals" is an old term from days when hashes were stored separately for leaves and internals. Nowadays it's hashes, not internals. I don't see any value to revert it